packages/mobile/src/components/user-list/UserCard.tsx

```diff
@@ -0,0 +1,85 @@
+import { useCallback } from 'react'
+
+import { SquareSizes, type ID } from '@audius/common/models'
+import { cacheUsersSelectors } from '@audius/common/store'
+import { formatCount } from '@audius/common/utils'
+import type { GestureResponderEvent } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import {
+  Avatar,
+  Divider,
+  Flex,
+  Paper,
+  Text,
+  type PaperProps
+} from '@audius/harmony-native'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+import { useProfilePicture } from '../image/UserImage'
+import { UserLink } from '../user-link'
+
+const { getUser } = cacheUsersSelectors
+
+const messages = {
+  followers: (count: number) => (count === 1 ? 'Follower' : 'Followers')
+}
+
+type UserCardProps = PaperProps & {
+  userId: ID
+  noNavigation?: boolean
+}
+export const UserCard = (props: UserCardProps) => {
+  const { userId, onPress, noNavigation, ...other } = props
+
+  const user = useSelector((state) => getUser(state, { id: userId }))
+  const navigation = useNavigation()
+
+  const handlePress = useCallback(
+    (e: GestureResponderEvent) => {
+      onPress?.(e)
+      if (noNavigation) return
+      navigation.navigate('Profile', { id: userId })
+    },
+    [onPress, noNavigation, navigation, userId]
+  )
+
+  const { source, handleError } = useProfilePicture(
+    userId,
+    SquareSizes.SIZE_480_BY_480
+  )
+
+  if (user === null) return null
+
+  const { handle, follower_count } = user
+
+  return (
+    <Paper border='default' onPress={handlePress} {...other}>
+      <Avatar
+        source={source}
+        onError={handleError}
+        accessibilityLabel=''
+        p='m'
+        pb='s'
+      />
+      <Flex ph='l' pb='s' gap='xs'>
+        <UserLink userId={userId} textVariant='title' />
+        <Text ellipses textAlign='center'>
+          @{handle}
+        </Text>
+      </Flex>
+      <Divider />
+      <Flex
+        pv='s'
+        backgroundColor='surface1'
+        alignItems='center'
+        borderBottomLeftRadius='m'
+        borderBottomRightRadius='m'
+      >
+        <Text size='s' strength='strong' style={{ lineHeight: 16 }}>
```

Comment:
I assume this line height is a corner case where our standard styles don't line up right?

Reply:
yeah, i have a comment in collectioncard, but ill add here as well